react/__tests__/components/SearchBar.test.js

```diff
@@ -35,10 +38,47 @@ describe('<SearchBar />', () => {
   })
 
   it('should have CSS handle searchBarIcon', () => {
-    const { container } = renderComponent()
-
+    const { container, debug } = renderComponent()
```

Comment:
All these `debug` are not being used? If so I don't think they should be declared.

Reply:
removed them 🙏